### DISCUSSION TITLE
Improve "records" window

### DISCUSSION THREAD

**Discussion by fheroes2bugs** [ihhub/fheroes2#7917] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

AI is getting stronger with every patch, so the same records in new version will be harder.
I played a lot of M maps so my XL maps records are gone forever and I can not compete myself in these maps...

### Describe the possible solution

I suggest to add some features to records windows:
1. Button to clear records
2. Make every record to be saved forever, so 2 buttons with up and down arrows.
3. Chosen difficulty. I can watch the size of the map, the difficulcty of the map, but chosen difficulty is gone forever, that is sad.

![records screen improvement](https://github.com/ihhub/fheroes2/assets/145407420/4f01c201-16ba-4bb9-bebf-cc11b249dce8)

PS I understand, that this window needs to be redrawn, it is not easy. So perhaps some designer will pick this issue in a year or two...

**Comment by ihhub** [comment]
I am moving this issue into the discussion section as the proposed ideas will not work for all versions of the game (languages) and it requires more than adding stuff into the existing window.

**Comment by fheroes2bugs** [comment]
It seems like up and down arrows could be added to every version, no matter what language it is...
